go.mod

```diff
@@ -36,4 +36,6 @@ require (
 	nhooyr.io/websocket v1.8.3
 )
 
+replace github.com/piprate/json-gold => github.com/trustbloc/json-gold v0.3.1-0.20200414173446-30d742ee949e
```

Comment:
we don't need it

Reply:
Yes,  aries-js-worker & other apps are using replace

Reply:
It's replaced to support unit tests.